packages/@pollyjs/core/tests/unit/polly-test.js

```diff
@@ -256,23 +271,35 @@ describe('Unit | Polly', function() {
     });
 
     it('.pause()', async function() {
```

Comment:
Should add a test asserting on the behavior of _order when a pause and play occurs with requests in between.  Will prevent a regression but also solidify expected behavior.